### DISCUSSION TITLE
pnpm support?

### DISCUSSION THREAD

**Discussion by RetrogradeDev** [vercel/turborepo#9250] [Help]
### Summary

Will there be pnpm support? This works great with npm, but it throws "module notfound" errors when I'm using pnpm as package manager.

**Comment by anthonyshew** [comment]
We support pnpm 9+, per our [Support Policy](https://turbo.build/repo/docs/getting-started/support-policy#package-managers). Is this an error that Turborepo is throwing or from your code?

**Reply by RetrogradeDev** [reply]
It's my code, it can't import a single package installed with pnpm (with next.js)

**Reply by anthonyshew** [reply]
Can you share a reproduction?

**Reply by RetrogradeDev** [reply]
Create a new next js project, run 'pnpm dlx shadcn@latest add calendar', import the competent somewhere and run 'next dev --turbo'

**Reply by anthonyshew** [reply]
That doesn't seem like it involves Turborepo. Are you meaning you're having problems with Turbopack?

**Reply by RetrogradeDev** [reply]
The next documentation redirected me to the Turborepo page, I excepted it to support the same package managers.

**Reply by anthonyshew** [reply]
Please check the Support Policy linked above that shows our pnpm support. The majority of folks using Turborepo are using pnpm.

Can you share where you found that link and a full reproduction of your case?

**Reply by RetrogradeDev** [reply]
The page where I found the link: [https://nextjs.org/docs/architecture/turbopack](https://nextjs.org/docs/architecture/turbopack)
My source code: [https://github.com/programORdie2/time-traveler](https://github.com/programORdie2/time-traveler)

**Reply by anthonyshew** [reply]
This repository is for Turborepo, but the linked you've provided isn't to a Turborepo. It seems like you're trying report an issue for Turbopack. Could you [open an issue in the Next.js repository](https://github.com/vercel/next.js/issues/new/choose), where the Turbopack team is currently working?
